v19.2/delete.md

```diff
@@ -109,52 +131,63 @@ To view how the index hint modifies the query plan that CockroachDB follows for
 
 For examples, see [Delete with index hints](#delete-with-index-hints).
 
-## Examples
+## Delete "expired" data
```

Comment:
This is going to perform a scan with the delete. This can be devastating to performance.  I would suggest an alternate 2 step approach.

First, issue a 

```sql
SELECT primaryKey 
  FROM tableName 
 WHERE timestamp < current_date() - INTERVAL '1 MONTH'
 LIMIT 10000 
 AS OF SYSTEM TIME -5s; 
```
(or follower reads if possible)

Then take the results and issue a delete

```sql
DELETE FROM tableName WHERE primaryKey IN [list of keys];
```

This minimizes contention from the scan and maximizes the speed of the deletes.